### DISCUSSION TITLE
Skin editor scenes buttons don't work

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#18755] [Q&A] [answered]
The buttons to switch between the gameplay and song select screens don't work. The game doesn't seem to give any errors, so I'm not sure if I missed something in the changelogs or if it's a real problem.

For reference: 
![buttons](https://user-images.githubusercontent.com/43073074/174439238-cee53d3c-323f-4474-9c9d-6887b4317366.gif)

**Comment by Gabixel** [comment]
I've updated the op to show the problem in detail (and apologize for writing the description too vaguely earlier 😅)

**Comment by frenzibyte** [comment, marked as answer]
Tracking in #18758
